spec/lib/hubspot/company_properties_spec.rb

```diff
@@ -43,7 +43,7 @@
       context 'with included groups' do
         cassette 'properties_in_groups'
 
-        it 'should return properties for the specified group[s]' do
+        xit 'should return properties for the specified group[s]' do
```

Comment:
Was this test failing previously? It looks like it's been changed from running to pending.

Reply:
Good question. I initially added the pending flag because updating RSpec to 3.x revealed that this test is not running the expectation. This became obvious because RSpec didn't complain that `be_true` is deprecated and needs to be replaced with `be true`.

I just updated the commit to remove the pending status but I did create a Trello ticket to address this test: [CompanyProperties: Test expectation isn't being executed](https://trello.com/c/zp5S0QH9). I'm tackling this now and will push up a separate PR :)